packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -480,13 +478,13 @@ class _BottomNavigationTile extends StatelessWidget {
               unselectedIconTheme: unselectedIconTheme ?? bottomTheme.unselectedIconTheme,
             ),
             _Label(
-              colorTween: colorTween,
+              colorTween: colorTween!,
               animation: animation,
               item: item,
-              selectedLabelStyle: selectedLabelStyle ?? bottomTheme.selectedLabelStyle,
-              unselectedLabelStyle: unselectedLabelStyle ?? bottomTheme.unselectedLabelStyle,
-              showSelectedLabels: showSelectedLabels ?? bottomTheme.showUnselectedLabels,
-              showUnselectedLabels: showUnselectedLabels ?? bottomTheme.showUnselectedLabels,
+              selectedLabelStyle: selectedLabelStyle,
+              unselectedLabelStyle: unselectedLabelStyle,
```

Comment:
fyi @HansMuller not sure if this was the intended design but this does match what we had before. Maybe we should make selectedLabelStyle and unselectedLabelStyle nullable in some future PR?

Reply:
This looks like an obvious error for which I assume we had no test coverage. This widget is supposed to be "conventional", non-null style parameters override defaults valued computed by the widget.

CC @johnsonmh

Reply:
Thanks for the catch, this PR should fix the bug and make desired nullability more clear: https://github.com/flutter/flutter/pull/67342